src/components/settings/SettingsSSOTab/steps/NewSSOConfigAuthorizeStep.tsx

```diff
@@ -65,7 +65,7 @@ const SSOConfigAuthorizeStep = () => {
    * Built using enterprise slug and uuid
    */
   const linkToDownloadMetadataXML = formFields?.spMetadataUrl
-    || `${configuration.EDX_ACCESS_URL}/samlp/metadata?connection=${enterpriseSlug}-${formFields?.uuid}`;
+    || `${configuration.EDX_ACCESS_URL}/samlp/metadata?connection=${enterpriseSlug.toLowerCase()}-${formFields?.uuid}`;
```

Comment:
Good catch!!
Optional chaining operator might fix the pipeline failure (`enterpriseSlug?.toLowerCase()`)